data_steward/utils/bq.py

```diff
@@ -93,7 +109,11 @@ def create_tables(client, project_id, fq_table_names, exists_ok=False):
     successes = []
     failures = []
     for table_name in fq_table_names:
-        schema = get_table_schema(table_name.split('.')[2])
+        if fields:
+            schema = fields
+        else:
+            schema = get_table_schema(table_name.split('.')[2])
+
```

Comment:
So, this might create a future bug.  It works for the case of building a single table.  I would opt to send the fields to `get_table_schema` to make sure the tables are built with the same schema fields when defined from a json file or from fields provided at run time.
Another concern is applying the same schema to multiple tables in an input table list.  You want to iterate through the fields list in the same fashion as the `fq_table_names`.

Reply:
This has been updated, please take a look.